src/components/keep_aspect_ratio.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import classNames = require('classnames');
+
+export interface keepAspectRatioProps {
+    /** the content to be sized appropriately  */
+    content: JSX.Element;
```

Comment:
would be better to pass in the element as the children

Reply:
I agree. I tried it, didn't work well unfortunately.
I would have to add a class to the child, to avoid using a descendant selector in CSS. It worked, but was very easy to break by accident, and could produce unexpected results.

Reply:
Changed it to work like that.